quinn-proto/src/streams.rs

```diff
@@ -423,6 +442,16 @@ impl Send {
         }
     }
 
+    /// Returns whether the stream was unblocked
+    pub fn raise_max_data(&mut self, offset: u64) -> bool {
```

Comment:
Naming nit: not sold on the use of `raise` here. Can we do `increase` or `bump` or `extend` or something else? It might be my past with Python that's confusing me. I also [looked it up](https://www.merriam-webster.com/dictionary/raise) and found that most highly ranked meanings don't seem to match this usage well.

Reply:
Good catch; in particular in software "raise" often refers to error conditions!

Reply:
Yeah, that's what I was referring to with my Python experience: in Python, `raise` is used to throw exceptions.